support/packages/local-npm/README.md

```diff
@@ -0,0 +1,213 @@
+# @liferay/local-npm
+
+`@liferay/local-npm` is a private package internal to this monorepo that
+provides a CLI tool to leverage local npm repos (like 
+[verdaccio](https://verdaccio.org/)) to publish local versions of development
+packages to be able to test them before the public release is made.
+
```

Comment:
:thinking: It doesn't setup verdaccio, but uses it...

Maybe:

> `@liferay/local-npm` [...] provides a CLI tool to be able to use local npm repos (like 
> [verdaccio](https://verdaccio.org/)) to publish local versions of development
> packages so that they can be tested before the public release is made.

:thinking:

Reply:
Yeah thats a good change. I hadn't gotten to the "how to use" part yet, so I was sort of guessing that it started it for you lol